### PR TITLE
add memoising for layer info

### LINKED ISSUE #50: Create an overview of all the data?
I.e. a searchable table of all services & layers, e.g. as a regularly updated article in the pkgdown website.

### PR DESCRIPTION
**Pull request by maelle**
Fix #50

(so no need for an article in the end :-) )

### CODE REVIEW

File: R/info.R [hunk 1]
```diff
@@ -42,12 +26,34 @@ emodnet_get_wfs_info <- function(wfs = NULL, service = NULL, service_version = "
...
-#' @inheritParams emodnet_get_layers
+#' Get WFS available layer information
+#'
+#' @rdname emodnet_get_wfs_info
+#' @param wfs A `WFSClient` R6 object with methods for interfacing an OGC Web Feature Service.
+#' @inheritParams emodnet_init_wfs_client
+#' @importFrom rlang .data
+#' @return a tibble containg metadata on each layer available from the service.
+#' @describeIn emodnet_get_wfs_info Get info on all layers from am EMODnet WFS service.
+#' @details To minimize the number of requests sent to webservices,
```
Comment (maelle) [inline R/info.R:37]: docs of the memoise behaviour here.

**Comment by annakrystalli** [comment]
Awesome! 

I'd like to start merging some PRs. Maybe this is a good one to start with if at least the mac & windows tests pass?

As this is user facing, should we add it to `NEWS.md` and bump the version?

_P.S. I royally messed up semantic versioning with this 😭 I know_

**Comment by maelle** [comment]
Should we use fledge? https://cynkra.github.io/fledge/

**Comment by annakrystalli** [comment]
Fledge looks super interesting! I'm tempted to implement alongside a `CONTRIBUTING.md` to ensure I am fully familiar with it's use and it's documented.

Let's open an issue about it for now, Is that cool?

**Comment by maelle** [comment]
Added a news item and opened an issue :-)

**Review by annakrystalli — APPROVED** [review]
LGTM. Maybe worth rerunning the tests?

**Pull request merged**